start.sh

```diff
@@ -138,15 +138,21 @@ if [ "${DEFAULT_SNAPSHOT}" == "MUST_PROVIDE_SNAPSHOT" ]; then
     echo "ERROR: Please specify a valid snapshot tag to continue."
     exit 2
 fi
-if [[ ! $DEFAULT_SNAPSHOT == 1.0.0-* ]]; then
-    echo "ERROR: invalid SNAPSHOT format... snapshot must begin with '1.0.0-'"
+
+echo "if [[ (! $DEFAULT_SNAPSHOT == 1.0.0-*) && ("$DOWNSTREAM" != "true") ]];"
+if [[ (! $DEFAULT_SNAPSHOT == 1.0.0-*) && ("$DOWNSTREAM" != "true") ]]; then
```

Comment:
is there a case that downstream GA is version `1.0.0`, or `1.0.1` only ? I know we currently have `1.0.0-RC2...RC3` working towards a release version of `1.0.0`?

Reply:
Downstream deploys from acm-d are manually mirrored to quay under a tag of our choosing, I think the "v<version>-RC#" moniker was decided upon by @tpouyer, are we going to tag a 1.0.0 upstream in acm-d Tim?